pkgs/default.nix

```
@@ -32,7 +32,23 @@
           // extra))
         .${wine};
     in {
-      inherit (inputs.umu.packages.${system}) umu;
+      umu = self.lib.mkDeprecated "warn" config.packages.umu-launcher {
+        name = "umu";
+        target = "package";
+        date = "2025-02-04";
+        instructions = ''
+          This package has been renamed to `umu-launcher` to
+          match the package name in nixpkgs.
+        '';
```

This really is a nit-pick, (sorry):

This string will be printed as-is to console output, so line wrapping should match the intended output _not_ necessarily conform to preferred code style.

This is unlike (e.g.) markdown option descriptions, where line-breaks are usually ignored and converted to normal spaces when rendering markdown to HTML.

```suggestion
        instructions = "This package has been renamed to `umu-launcher` to match the package name in nixpkgs.";
```

One work-around if you really want to reduce the line-length is to use string concatenation:

```nix
instructions =
  "This package has been renamed to `umu-launcher` "
  + "to match the package name in nixpkgs.";
```

---

I'm personally not concerned about it. This matches the existing message for `proton-ge` and ideally they will simply change the package name and never see the message again